cli/provider/cmd.go

```diff
@@ -219,6 +219,7 @@ func (c *CmdConfigurator) AddFlags(cmd *cobra.Command) error {
 	case "keploy":
 		cmd.PersistentFlags().Bool("debug", c.cfg.Debug, "Run in debug mode")
 		cmd.PersistentFlags().Bool("disableTele", c.cfg.DisableTele, "Run in telemetry mode")
+		cmd.PersistentFlags().Bool("enableANSIColor", c.cfg.EnableANSIColor, "Enable ANSI color in logs")
```

Comment:
The changes are working correctly, just change the name of this flag to "enableANSI". Currently the name feels too long to type for a flag.

Reply:
refactored